site/src/components/SidebarAndContent.scss

```diff
@@ -65,6 +66,16 @@ $tableOfContents-width: $ca-grid * 10;
   }
 }
 
+.healthContent {
+  margin-top: 2.25rem;
+
+  // Sorry for the hax but there's no way for me currently to add margin bottom
+  // to each tag without creating a parent element for each.
+  div[class^="Tag--"] {
```

Comment:
I'd prefer the parent element approach as we can't always guarantee the classes on components won't change (e.g. if we changed the name of the Tag component to something else, we'd probably rename the classes too)

Reply:
F